docs/extensions.adoc

```diff
@@ -155,6 +155,29 @@ Groovy has the `groovy.transform.NotYetImplemented` annotation which is similar
 def "not implemented yet"() { ... }
 ----
 
+=== PendingFeatureIf
+
+To conditionally indicate that a feature is not fully implemented, and should not be reported as an error you can annotate
+it as `spock.lang.PendingFeatureIf` and include a precondition similar to `IgnoreIf` or `Requires`
+
+If the conditional expression _passes_ it behaves the same way as `PendingFeature` - otherwise it does nothing.
+
+(For instance, annotating a feature as `@PendingFeatureIf({false})` effectively does nothing - but annotating it as
+ `@PendingFeatureIf({true}) behaves the same was as if it was marked as `@PendingFeature`)
+
+[source,groovy]
+----
+@PendingFeatureIf({ os.windows })
+def "I'm not yet implemented on windows, but I am on other operating systems"() { ... }
+----
+
+[source,groovy]
+----
+@PendingFeatureIf({ System.getProperty("targetEnvironment").startsWith("production") })
```

Comment:
Take a look at `PreconditionContext` we have a shorthand for that `sys`, and I would just use `== 'prod'`

And just merge those two blocks